packages/itwinui-react/src/core/Table/TableExpandableContentMemoized.tsx

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import * as React from 'react';
+import { Box } from '../../utils/index.js';
+import type { VirtualItem } from '@tanstack/react-virtual';
+import type { PolymorphicForwardRefComponent } from '../../utils/index.js';
+
+type TableExpandableContentProps = {
+  virtualItem?: VirtualItem<Element>;
+  isDisabled: boolean;
+  children: React.ReactNode;
+  isSelected?: boolean;
+};
+
+const TableExpandableContent = React.forwardRef((props, ref) => {
+  return (
+    <Box
+      className={cx('iui-table-row', 'iui-table-expanded-content')}
+      style={{
+        flex: '0 0 auto',
+        minWidth: '100%',
+        ...(props.virtualItem != null
+          ? { transform: `translateY(${props.virtualItem.start}px)` }
```

Comment:
Virtualization doesn't currently work with expanded content, right? So let's remove all virtualization-specific code from this component.

Reply:
I have removed virtualization-relating code.